package/kernel/lantiq/ltq-deu/src/ifxmips_sha1_hmac.c

```diff
@@ -400,8 +400,6 @@ static struct shash_alg ifxdeu_sha1_hmac_alg = {
     .update             =       sha1_hmac_update,
     .final              =       sha1_hmac_final,
     .setkey             =       sha1_hmac_setkey,
-    .init_tfm           =       sha1_hmac_init_tfm,
-    .exit_tfm           =       sha1_hmac_exit_tfm,
```

Comment:
Is this driver not compiling with kernel 5.4 without this patch any more? 
OpenWrt should compile on every commit, so that a git bisect works. 

If compilation with kernel 5.4 was broken with some previous commit from this series please change the previous patch.

Reply:
I started with 5.10 and then it was discovered that it does not work with 5.4, so I will merge the fixing commit with the original one.

Reply:
ok thanks for merging  this together